BuildAndTest.proj

```diff
@@ -12,6 +12,8 @@
   <PropertyGroup>
     <RoslynSolution>$(MSBuildThisFileDirectory)Roslyn.sln</RoslynSolution>
     <RoslynSolution Condition="'$(PublicBuild)' == 'true'">$(MSBuildThisFileDirectory)RoslynLight.sln</RoslynSolution>
+    <CscCoreProj>$(MSBuildThisFileDirectory)\src\Compilers\CSharp\csc-core\csc-core.csproj</CscCoreProj>
```

Comment:
Please use PascalCasing for project names instead of dashes.  Nothing else in our repo uses dashes as separators.  This would be a deviation from the rest of the tree.